### ISSUE TITLE
Scaffolding fails when upgrading existing @angular/cli project that uses yarn

### ISSUE THREAD

**Issue by jsonberry** [nrwl/nx#701]
Not sure if this should be a feature request or a bug report.

## Versions
`@angular/cli`: ~6.1.0
`@nrwl/schematics`: ~6.2.0

## Steps to reproduce
```
yarn global add @angular/cli // (6.1.0 - 6.1.3, different error with versions prior to 6.1.0)
yarn global add @nrwl/schematics // (6.2.0)

ng config -g cli.packgeManager yarn // <-- the main issue

ng new foo
ng add @nrwl/schematics
```
## Expected
Nx workspace scaffolds correctly in existing Angular CLI project

## Actual
Scaffolding fails and this error is thrown: `Cannot read property 'kind' of undefined`

---
Note: this isn't related to installing packages globally with `yarn`, but it is related to `@angular/cli` projects that are running `yarn` as the package manager, they consistently fail. 

This became easily noticeable when using the new Angular Console and adding `@nrwl/schematics` via the CLI extensions section, but obviously fails when using  `ng add` outside of the Console.

**Comment by JRinkoff** [comment]
Ran into the same issue as @jsonberry 

Using:
`@angular/cli: 6.1.2`
`@nrwl/schematics: 6.2.0`

Resolved by upgrading typescript from 2.7.2 to 2.9.2:
`yarn global remove typescript`
`yarn global add typescript@~2.9.2`
`yarn upgrade --dev typescript@~2.9.2`

**Comment by FrozenPandaz** [comment]
This issue is caused by a mismatch in `typescript` versions and should be fixed in the latest version of Nx (`6.4.0-beta.3`)

**Comment by matheo** [comment]
@FrozenPandaz thanks a lot for writing up the cause!
I installed `rxjs-tslint` and it had a wide open dependency to `typescript@>=2.8.3` so it screwed it all installing `typescript@3.5.1` when the Angular project is working with `3.4.5`
